tests/integration/devfile/cmd_dot_devfile_test.go

```diff
@@ -0,0 +1,168 @@
+package devfile
+
+import (
+	"fmt"
+	"path/filepath"
+	"regexp"
+	"strings"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	//We continued iterating on bracket pair guides. Horizontal lines now outline the scope of a bracket pair. Also, vertical lines now depend on the indentation of the code that is surrounded by the bracket pair.. "github.com/onsi/gomega"
+	"github.com/openshift/odo/tests/helper"
+)
+
+var _ = Describe("Test suits to check .devfile.yaml compatibility", func() {
+	var cmpName string
+	var commonVar helper.CommonVar
+
+	BeforeEach(func() {
+		commonVar = helper.CommonBeforeEach()
+		cmpName = helper.RandString(6)
+		helper.Chdir(commonVar.Context)
+
+		odoArgs := []string{"catalog", "list", "services"}
+		operators := []string{"redis-operator"}
+		for _, operator := range operators {
+			helper.WaitForCmdOut("odo", odoArgs, 5, true, func(output string) bool {
```

Comment:
This operation can be quite long, please place this block in a BeforeEach where it is needed only, and not the more englobing one (in `When("Creating a nodejs component and replace devfile.yaml with .devfile.yaml"` I think).